bin/all_sky_search/pycbc_plot_kde_vals

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+import h5py, argparse
+import matplotlib.pyplot as plt
+from matplotlib.colors import LogNorm
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--signal-file')
+parser.add_argument('--template-file', required=True)
+parser.add_argument('--param', nargs='+', required=True, help='mchirp, eta, chi_eff')
+parser.add_argument('--plot-type', choices=['kde_vs_param', 'param_vs_param'])
+parser.add_argument('--which-kde', choices=['signal_kde','template_kde', 'ratio_kde'])
+parser.add_argument('--plot-dir', required=True)
+parser.add_argument('--verbose', action='count')
+args = parser.parse_args()
```

Comment:
Can you add some input checking so that if the user chooses kde_vs_param the length of args.param should be constrained to be exactly 1, if they choose param_vs_param the length should be exactly 2.
The basic point is that there should not be any options given to the code that have no effect. So e.g. in the kde_vs_param case any --param options beyond the first one are not used and would be meaningless.